templates/login/login_core.html.twig

```diff
@@ -1,10 +1,15 @@
-{% set srOnly = (showLabels == true) ? "" : "sr-only" %}
 <!DOCTYPE html>
 <html lang="en">
 <head>
     {% if displayGoogleSignin %}
         <meta name="google-signin-client_id" content="{{ googleSigninClientID|attr }}">
     {% endif %}
+   <script src="https://accounts.google.com/gsi/client" async defer></script>
+    <div id="g_id_onload"
+         data-client_id="{{ googleSigninClientID }}"
+         data-callback="onSignInSuccess" data-auto_prompt="false"> 
+   </div>
```

Comment:
shouldn't this block of code be within the above if conditional in line 4?
(ie. only shows `if displayGoogleSignin`)